### PR TITLE
wip : Fix workflow list request to return closure data aswell

### PR DESCRIPTION
**Pull request by pmahindrakar-oss**
Signed-off-by: Prafulla Mahindrakar <prafulla.mahindrakar@gmail.com>

# TL;DR
Return the closure in the workflow list response aswell.
This might have been omitted for a purpose may be due to size of the closure data.
If not this then can we have the client get the latest version using one call and then call get workflow with it.

But i do see that we return the entire objects in case of launch plan list request so fixing this in the admin would be ideal.

## Type
 - [X] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

Unit tests  pending

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/flyteorg/flyte/issues/1053

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

**Comment by katrogan** [comment]
@pmahindrakar-oss is there a use-case for this or are you just fixing for consistency's sake? the closure can indeed be very big so it would be ideal if flytectl can call a get in the background transparently

**Comment by pmahindrakar-oss** [comment]
@katrogan its more to make it consistent with launch plan behavior , but i agree if the payload is going to be huge then flytectl can call the get transparently . Will send PR to make launchplan consistent with this

**Comment by katrogan** [comment]
@pmahindrakar-oss what do you mean by making launch plan consistent with this?
